packages/scripts/README.md

```diff
@@ -0,0 +1,125 @@
+
+### clone (git)
+
+Given the add-on remote git repository it pulls and configures it into the vanilla project generated by the script.
+
+`npx -p @plone/scripts addon clone [options] <source> [destination]`
+
+    Usage: addon clone [options] <source> [destination]
+
+    clone a repository into a newly created directory
+
+    Options:
+      -p, --private          set if the repo is private, then GITHUB_TOKEN is used
+      -b, --branch <branch>  set the repo branch, defaults to main
+      -c, --canary           downloads latest Volto canary (alpha) version
+      -h, --help             display help for command
+
+This next command, would download the `volto-blocks-grid` add-on from its git repo, branch `main` and will generate a project with the latest Volto canary (alpha) version.
+
+```shell
+npx -p @plone/scripts addon clone https://github.com/kitconcept/volto-blocks-grid.git --branch main --canary
+```
+
+This will create a directory named `addon-testing-project` and will bootstrap a new project using Volto's standard project generator.
+It will adjust the configuration of this project to setup the add-on in the project using `mrs-developer` and the git URL given to fetch the add-on contents.
+You can specify the branch to be used, if the project should use the latest alpha available.
+There is an option for private repos as well, in that case, it will use the `GITHUB_TOKEN` present in your environment variables to fetch it.
+
+After this, as a developer you can use the usual project commands to run tests (unit, linting, acceptance) inside the generated `addon-testing-project`.
+You can configure the CI of your choice for automated testing, you can take a look at how it's done in: https://github.com/kitconcept/volto-blocks-grid/tree/main/.github/workflows
+
+The idea is to issue commands inside the generated `addon-testing-project` project and do your checks.
+Take special care on how to pass down to the `npx` command the current PR branch depending on your CI system this might be different.
+
+### clone local
+
+You can also clone the local add-on using:
+
+```shell
+npx -p @plone/scripts addon clone .
+```
+
+This only works if you execute the command from the root of your add-on directory.
+
+### consolidate
+
+While developing, you might have done changes inside the generated project, and you most probably want to consolidate them, back into the root of the repository.
+By running this script, it copies over from `addon-testing-project/src/addons/<my-addon>` to the root of your repository.
+
+It should be run at the root of the add-on, and it gets an optional `source` argument in case you have specified a directory other than `addon-testing-project`.
+
+`npx -p @plone/scripts addon consolidate --help`
+
+    Usage: addon consolidate [options] [source]
+
+    Consolidate a cloned project
+
+    Options:
+      -h, --help  display help for command
```

Comment:
See above note (line 83) regarding how to put this entire help output in a `console` code block.

Reply:
Done